api/v1alpha1/storage_webhook.go

```diff
@@ -112,13 +112,25 @@ func (r *Storage) ValidateCreate() error {
 		}
 	}
 
+	if r.Spec.Monitoring != nil {
+		if !isMonitoringCRDExists(manager) {
+			return fmt.Errorf("Monitoring CRDs (`monitoring.coreos.com/v1/servicemonitor`) do not exists, please check if prometheus CRDs installed correctly")
```

Comment:
The same issue here, please make the error message consistent. Maybe even make this whole new block (115-120) a function, to not duplicate it four times?